wp-job-manager-functions.php

```diff
@@ -1707,3 +1707,29 @@ function job_manager_get_salary_unit_options( $include_empty = true ) {
 	 */
 	return apply_filters( 'job_manager_get_salary_unit_options', $options, $include_empty );
 }
+
+/**
+ * Check if user can submit job listings.
+ *
+ * @return bool
+ * @since $$next-version$$
+ */
+function job_manager_user_can_submit_job_listing() {
+	$can_submit       = true;
+	$submission_limit = get_option( 'job_manager_submission_limit', '' );
+	$job_count        = job_manager_count_user_job_listings();
+
+	if ( '' !== $submission_limit &&
```

Comment:
You can set the $can_submit variable only once with:

```
	$can_submit = '' === $submission_limit || $submission_limit > $job_count;
```

Reply:
Thanks @gkaragia !  Done in https://github.com/Automattic/WP-Job-Manager/pull/2362/commits/cc64267ca8d43a68a744acdba11890cc6604017a